src/Adapter/AdapterServiceFactory.php

```diff
@@ -12,7 +12,6 @@ class AdapterServiceFactory implements FactoryInterface
      * Create db adapter service
      *
      * @param string $requestedName
-     * @param array $options
      * @return Adapter
      */
     public function __invoke(ContainerInterface $container, $requestedName, ?array $options = null)
```

Comment:
Do you think we should go ahead and update the typing here for Psr\Container\ContainerInterface and remove the v2 methods?